src/data/machines.js

```diff
@@ -40,6 +40,10 @@ export const machineTypes = _.map(({ price, preemptiblePrice, ...details }) => (
 export const monthlyStoragePrice = 0.04 // from https://cloud.google.com/compute/pricing
 export const storagePrice = monthlyStoragePrice / 730 // per GB hour using 730 hours per month from https://cloud.google.com/compute/pricing
 export const dataprocCpuPrice = 0.01 // dataproc costs $0.01 per cpu per hour
+export const ephemeralExternalIpAddressPrice = { // per hour in dollars for Iowa (us-central1) region per https://cloud.google.com/vpc/network-pricing#ipaddress
+  standard: 0.004,
+  preemptible: 0.002
```

Comment:
I think this is already accounted for in lines 35-36 above.

Reply:
I had missed this somehow. Thanks for point it out! 

If I'm reading the pre-existing code correctly, it's not accounting for VM status. Ephemeral external IP address cost is incurred only when the VM is running. Trying to see if I can find a clever way to adjust that. If you have any thoughts, let me know.

Reply:
Okay, I pushed a commit that adds IP cost to runtimes only when they're running. But Galaxy cost calculation flow is different and currently it's adding the IP cost whether a Galaxy app is running or stopped. I may need help with figuring out how to make app status available to `getGalaxyCost()` or finding another way to make the distinction.

Reply:
Paired with Emily and got help with factoring status into cost calculations.